cucumber/authenticators_gce/features/step_definitions/authn_gce_steps.rb

```diff
@@ -14,19 +15,44 @@
     raise "Incorrect annotation name: '#{annotation_name}', expected: service-account-id|service-account-email|project-id|instance-name"
   end
 
-  set_annotation_to_resource("#{annotation_name}", annotation_value)
+  set_annotation_to_resource("authn-gce/#{annotation_name}", annotation_value)
 end
 
-Given(/^I obtain a GCE identity token in "([^"]*)" format with audience claim value: "([^"]*)"$/) do | audience, format |
+# Sets all GCE annotations
+Given(/^I set GCE annotations to host "([^"]*)"$/) do | hostname |
+  i_have_a_resource "host", hostname
+
+  set_annotation_to_resource("authn-gce/service-account-id", gce_service_account_id)
+  set_annotation_to_resource("authn-gce/service-account-email", gce_service_account_email)
+  set_annotation_to_resource("authn-gce/project-id", gce_project_id)
+  set_annotation_to_resource("authn-gce/instance-name", gce_instance_name)
+end
+
+# Runs a curl command in a remote machine
+Given(/^I obtain a GCE identity token in (full|standard) format with audience claim value: "([^"]*)"$/) do | format, audience |
   gce_identity_access_token(
     audience: audience,
     token_format: format
   )
 end
 
-Given(/I authenticate with authn-gce using Google identity token/) do
+# Authenticates with Conjur GCE authenticator
+Given(/I authenticate with authn-gce using (no |an empty |invalid )?token and (non )?existing account/) do | token_state, non_existing_account |
```

Comment:
i think that `an` is redundant. but if you do use it, then use it also for `invalid`.

Reply:
I agree.